client/src/main/java/org/asynchttpclient/DefaultAsyncHttpClient.java

```diff
@@ -255,6 +257,13 @@ public EventLoopGroup getEventLoopGroup() {
         return channelManager.getEventLoopGroup();
     }
 
+    public ClientStats getClientStats() {
```

Comment:
Performing the computation should be the responsibility of `ChannelManager`, not `DefaultAsyncHttpClient`.